.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -100,15 +100,15 @@ else
   fi
 fi
 
-if ! [[ "$HARDWARE_ARCH" = aarch64 && -n "$CPP_CROSS_COMPILE" ]] && [[ $TEST_OUTCOME -eq 0 ]] ; then 
-  buildkite-agent artifact upload "build/distributions/*.zip"
-fi
-
 if [[ -z "$CPP_CROSS_COMPILE" ]] ; then 
-  OS=$(uname -s | tr "A-Z" "a-z")
-  TEST_RESULTS_ARCHIVE=${OS}-${HARDWARE_ARCH}-unit_test_results.tgz
-  find . -path  "*/**/ml_test_*.out" -o -path "*/**/*.junit" | xargs tar cvzf ${TEST_RESULTS_ARCHIVE}
-  buildkite-agent artifact upload "${TEST_RESULTS_ARCHIVE}"
+  if [[ $TEST_OUTCOME -eq 0 ]] ; then 
+    buildkite-agent artifact upload "build/distributions/*.zip"
```

Comment:
By moving this here will we still upload cross compiled Intel macOS artifacts?

Reply:
> 

Hmm no. In hindsight I think you're right, this file was better as it was. I'll revert these changes and stop playing with it.